app/src/main/java/com/animal_villa/dto/Player.kt

```diff
@@ -1,4 +1,14 @@
 package com.animal_villa.dto
+/**
+ * A class Player is noun class Data Transfer Object.
+ * @property status the status for every Player : Int
+ * @property money the money for every Player : Int
+ * @property energy the energy for every Player : Int
+ * @property currentLocation the currentLocation for every Player : Int
+ */
+data class Player(var status : Int, var money : Int, var energy : Int, var currentLocation : Int) {
+    /**
+     * Contains all the methods for data class player
+     */
```

Comment:
Included KDoc to make code readable https://kotlinlang.org/docs/kotlin-doc.html
Implemented data class instead of class as it is a DTO with the purpose to hold data. https://kotlinlang.org/docs/data-classes.html
Initialized constructor variables accorrding to conventions, class headers https://kotlinlang.org/docs/coding-conventions.html#class-headers